### PR TITLE
DRUM added

### PR DESCRIPTION
**Pull request by ayushseth07**

### CODE REVIEW

**Comment by chicken-biryani** [comment]
Which issue does this deal with?

**Comment by chicken-biryani** [comment]
Add your project Title and description in index.md

**Review by harshita2216 — CHANGES_REQUESTED** [review]
Please add a Readme file with the following details - 

1. Introduction
2. Use
3. How to Setup
4. Tech Stack
5. Relevant Screenshots

**Comment by ayushseth07** [comment]
It deals with issue no. #73

**Review by Ayushparikh-code — APPROVED** [review]

File: DRUM/README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+- Download or clone the repository
+
+```
+git clone https://github.com/Ayushparikh-code/Web-dev-mini-projects.git
+```
+
+- Go to the directory
+- Run the index.html file
+- Start Using The Clock!!
```
Comment (chicken-biryani) [inline DRUM/README.md:21]: Please add images of the output.

File: DRUM/README.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# DRUM
+
+*This web app allows you to make your own music with all the beats that are present in the app. You can click on the alphabets or the images that are present on the screen and enjoy the beats. You can also use your keyboard keys to make the music.*
+>Technologies used:
```
Comment (chicken-biryani) [inline DRUM/README.md:4]: Put this in bold. ANd use a markdown beautifier.

File: DRUM/README.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+```
+git clone https://github.com/Ayushparikh-code/Web-dev-mini-projects.git
+```
+
+- Go to the directory
+- Run the index.html file
+- Start Using The DRUM
+## Relevant Screenshots of the application
+
+![drum](images/drumkit.PNG)
```
Comment (chicken-biryani) [inline DRUM/README.md:24]: THe screenshot is not working.

File: DRUM/index.html [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+                <audio class="sound" src="sounds\sound2.mp3"></audio>
+            </div>
+        </div>
+    </div>
+
+    <script src="app.js"></script>
+
+</body>
+
+</html>
```
Comment (chicken-biryani) [inline DRUM/index.html:72]: Use an HTML beautifier.

File: DRUM/style.css [hunk 5]
```diff
@@ -0,0 +1,211 @@
...
+    }
+    .keyboard-keys{
+        font-size:1.5rem;
+        top:-20px;
+    }
+    .header p
+    {
+        font-size: 1rem;
+    }
+}
```
Comment (chicken-biryani) [inline DRUM/style.css:211]: Use a css beautifier.

**Review by chicken-biryani — COMMENTED** [review]
Also add your project name and description to index.md

File: DRUM/README.md (hunk 3 above)
Comment (ayushseth07) [reply DRUM/README.md:24]: the screenshot is visible i guess.
image

Comment (chicken-biryani) [reply DRUM/README.md:26]: @ayushseth07 No it isn't. Please use a merged PR for reference.

**Pull request merged**